icaruscode/CRT/CRTDecoder/decodercrtjob.fcl

```diff
@@ -39,11 +40,13 @@ physics:
  producers:
  {
    crtdaq: @local::crtdaq_icarus
+   daqTrigger: @local::decodeTrigger
+
  }
 
  #define the producer and filter modules for this path, order matters, 
  #filters reject all following items.  see lines starting physics.producers below
- daq: [  crtdaq ]
+ daq: [  crtdaq , daqTrigger]
```

Comment:
If `crtdaq` needs `daqTrigger`, `daqTrigger` needs to be run before `crtdaq`:
```suggestion
 daq: [ daqTrigger, crtdaq ]
```

Reply:
Thanks Gianluca for this comment. Does the order of what is within the `producers:{}` bracket matter? Should I put the daqTrigger before the crtdaq line here as well?
` producers:
 {
   crtdaq: @local::crtdaq_icarus
   daqTrigger: @local::decodeTrigger

 }`